types/module/module.go

```diff
@@ -460,13 +460,6 @@ func (m *Manager) RegisterServices(cfg Configurator) error {
 			module.RegisterServices(cfg)
 		}
 
-		if module, ok := module.(appmodule.HasServices); ok {
-			err := module.RegisterServices(cfg)
-			if err != nil {
-				return err
-			}
-		}
```

Comment:
Why we don't need this anymore? I see you reverted some related changes, maybe this has to be reverted too?

Reply:
ah i think i didnt revert this, on it. thank you